migrations/sqls/20201028204537-followups-up.sql

```diff
@@ -5,6 +5,7 @@ CREATE TABLE IF NOT EXISTS followUps (
   description MEDIUMTEXT,
   dateOfContact DATETIME NOT NULL,
   appointmentDate DATETIME,
+  duration TIME DEFAULT '0:00:00' NOT NULL,
```

Comment:
Modifying a migration doesn't work, since the migration has already run on the production database. Instead, create a new migration that does `ALTER TABLE followUps ADD COLUMN`